src/clj/rems/css/styles.clj

```diff
@@ -96,14 +100,16 @@
                            :background-size :contain
                            :background-repeat :no-repeat
                            :background-position [[:center :center]]}]
-                         [:.logo
-                          {:height (u/px 150)}]))
+                         [:.logo {:height logo-height
+                                  :min-height logo-height}]))
```

Comment:
Why is this `:min-height` needed now?

Reply:
I believe I was tinkering with nav styles and arrived at a conclusion it was needed, but after a few iterations lost the sight of it. It neither fixes nor breaks anything.

Reply:
In that case the less there is of diff the better so we should remove it.